test/e2e/performanceprofile/functests/utils/images/images.go

```diff
@@ -13,7 +13,7 @@ func init() {
 	cnfTestsImage = os.Getenv("CNF_TESTS_IMAGE")
 
 	if cnfTestsImage == "" {
-		cnfTestsImage = "cnf-tests:4.14"
+		cnfTestsImage = "cnf-tests:4.19"
```

Comment:
We need a fix that does not hardcode the image version.

Reply:
That is actually hard to do here, the source has no way to find out the version of a build from a different repo. That is why the env var is here.

Reply:
this is the fallback default, it's a fair assumption to go in lockstop with the NTO branch/version. The existing env var can and should be used for fine tuning, leaving the hardcoded value to sane defaults.